site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -284,20 +305,33 @@ This indicates that Kickstart completed and provides a summary of the configurat
 
 You may also want to check out https://github.com/FusionAuth/fusionauth-example-scripts/tree/master/isolated-docker-setup[the Isolated Docker Setups] if you want the ability to rapidly stand up different versions and configurations of FusionAuth.
 
+If you want to test changes to your Kickstart file, you'll need to delete your volumes each time. Kickstart won't run except on a brand new install. If there is any data in the database, it won't proceed.
+
+[WARNING.warning]
+====
+This will delete all data in your docker instance.
+====
+
+[source,title="Deleting the volumes"]
+----
+docker-compose down -v
+----
+
 == Plugins
 
 You can mount a directory containing a plugin to your Docker container.
 
 Here are the steps to do so:
 
 * In the `volumes:` section of the `fusionauth` service, add `- ./plugins:/usr/local/fusionauth/plugins`.
-* Copy your plugin jar file, created link:/docs/v1/tech/plugins/[by following the instructions], to your local `plugins` directory.
-* Start your Docker file
+* Copy your plugin jar file, created link:/docs/v1/tech/plugins/[by following the instructions], to your `plugins` directory on the host.
+* `docker-compose up`
 
 The following is an example `docker-compose.yml` file configuring FusionAuth to scan for plugins at startup.
```

Comment:
See comment above - should this be a separate file in the fusionauth-containers repo (maybe the burden of support is too great here) or a separate include file?

Reply:
See my comment above, no real good way to do this. Alternatives are:

* keep the yaml files in the doc
* put it in fusionauth-containers, which currently only contains prod code, not examples
* put in fusionauth-contrib and pull them in there
* find some way to generate them by modifying the base file, perhaps with a compose override.

I'm not sure what the best option is today, so I'm going to kick the can a bit and go with option 1.